.changes/unreleased/Features-20240405-175733.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Features
+time: 2024-04-05T17:57:33.047963+02:00
+custom:
+  Author: SamuelBFavarin
+  Issue: "9747"
```

Comment:
The `body` section of the changelog yaml is missing.  It can be as simple as "Updated NoNodesForSelectionCriteria message to be more clear".  It's a required field and our changelog processing will break without it.